discojs/discojs-core/src/client/decentralized/base.ts

```diff
@@ -165,6 +165,7 @@ export abstract class Base extends ClientBase {
   async onRoundEndCommunication (
     updatedWeights: WeightsContainer,
     staleWeights: WeightsContainer,
+    _: WeightsContainer, // TODO: Implement Byzantine-Robust Aggregator in Decentralized setting
```

Comment:
is it needed?

Reply:
The parent class of `decentralized/base.ts` defines a third parameter `updatedMomentum` that is not used in decentralized setting, but we cannot avoid to put it in the signature because it is inherited from the parent class. However, it will be useful when implementing the Byzantine aggregator also in the decentralized setting.